test_schemas/en/test_theme_business.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
I have added this as we have schemas for a couple of the other themes (however health is missing), however we don't use them in any tests. It does raises the question with what we want to do with theme schemas and testing.

Reply:
I think we should just remove them if they don't have tests. They could be added in when needed with the appropriate tests.